src/database/models/Redirections.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  DataType,
+  Column,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  ForeignKey,
+} from "sequelize-typescript"
+
+import { Launches } from "@database/models/Launches"
+import { Site } from "@database/models/Site"
+import { User } from "@database/models/User"
+
+@Table({ tableName: "redirections", paranoid: true })
+export class Redirections extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @ForeignKey(() => Launches)
+  @Column
+  launchId!: number
```

Comment:
see above comment wrt [model associations](https://github.com/sequelize/sequelize-typescript#model-association)

Reply:
Wait no hmm, it should be separate, something like this:

```ts
@ForeignKey(() => Launches)
@Column
launchId!: number

@BelongsTo(() => Launches, {
  onUpdate: "CASCADE",
  onDelete: "CASCADE",
})
launch!: Launches
```

Side note: Just realised we should also use singular for the models, but plural for the actual database name.

Reply:
cc438e2

Reply:
Looks good now, thanks!